pkg/middleware/validation.go

```diff
@@ -45,31 +47,60 @@ func NewValidationMiddleware(validatorHook string) mux.MiddlewareFunc {
 	}
 }
 
+func maybeLogValidationReason(valid ValidationResponse, r *http.Request, mod string, version string) {
+	message := valid.ValidationMessage
+	if message.Reason != "" {
+		entry := log.EntryFromContext(r.Context())
+		entry.Warnf("Module %s:%s Reason: %s, Description: %s", mod, version, message.Reason, message.Description)
```

Comment:
As for `Reason: %s, Description: %s`, couldn't the server return that message formatting and so we would just log it out without having to restrict all Validation Hook Servers to follow the same exact shape? In other words we can just print the http response body as is and the validation server is responsible for formatting the message however it wants.

Reply:
Now just a simple message retrieved from the body https://github.com/gomods/athens/pull/1517/commits/933d4430dadc1cfd1b8a34d85162930e503434b3